README.md

```diff
@@ -90,15 +92,6 @@ Versions are auto incremented based on commit messages and SemVer (Major.Minor.P
 - `patch:` or `fix:` example `fix: url bug` 
   - this will increment the PATCH version eg 1.0.0 to 1.0.1. This will also happen by default.
 
-#### Build process
-
-1. Pushes to any branch will start the build process
-2. `.circleci/circle_trigger.sh` will check for any changes in our packages, and if builds have failed previously
-3. `circle_trigger.sh` will trigger a workflow via the API. It will pass the parameters model, runner, designer (bool) to the workflow.
-4. If there are any changes to a workspace, it will be built and tested.
-
-- If an upstream dependency, like model has changed, the downstream dependencies (runner, designer) will also be built and tested.
-
 #### Development environment
 
```

Comment:
Should we change this to the below?

When we merge a PR into the main branch it will trigger the development workflow. You can monitor this on the repository's [action tab](https://github.com/XGovFormBuilder/digital-form-builder/actions).